code/modules/modular_computers/computers/subtypes/dev_wristbound.dm

```diff
@@ -41,4 +37,4 @@
 	if(ishuman(user))
```

Comment:
If you *wanted*, you could move the identical parts of `attack_hand` and `AltClick` into their own proc. Since they only differ by a `..()`

But that's totally non-blocking and not that important.

Reply:
Mm, they have identical parts, but their end functions are different. I think obscuring that behind a proc to save 2 lines of code would just serve to make it more difficult to read.